### PR TITLE
Develop into main

### PR DESCRIPTION
**Pull request by bsatoriu**

### CODE REVIEW

File: devfiles/isce3/devfile/devfile.yaml [hunk 1]
```diff
@@ -55,8 +55,12 @@ components:
                     mountPath: /projects/shared-buckets
                     subPath: shared-buckets
                     mountPropagation: HostToContainer
+                  - name: s3fs-volume
+                    mountPath: /projects/triaged-jobs
+                    subPath: triaged-jobs
+                    mountPropagation: HostToContainer
               - name: s3fs
-                image: 'mas.ops.maap-project.org/root/che-sidecar-s3fs:master'
+                image: 'mas.dit.maap-project.org/root/che-sidecar-s3fs:master'
```
Comment (grallewellyn) [inline devfiles/isce3/devfile/devfile.yaml:63]: Should this be dit? @bsatoriu

Comment (bsatoriu) [reply devfiles/isce3/devfile/devfile.yaml]: Yes

Comment (grallewellyn) [reply devfiles/isce3/devfile/devfile.yaml]: How does ops get updated to the .ops. sidecar image?

**Review by grallewellyn — APPROVED** [review]
Successfully loaded maap-py in DIT and an image in ops build from the dit image

**Pull request merged**